packages/flutter_tools/test/integration.shard/test_data/hot_reload_websocket_test_common.dart

```diff
@@ -0,0 +1,189 @@
+// Copyright 2025 The Flutter Authors. All rights reserved.
```

Comment:
I'm assuming this may make sense in a future PR, but do we need to have a common file for this if there's only one test?